.circleci/config.yml

```diff
@@ -1,15 +1,18 @@
 version: 2.1
 orbs:
   ruby: circleci/ruby@1.1.1
+  buildevents: honeycombio/buildevents@0.5
+  carwow: carwow/carwow-orb@0.4
 
 jobs:
   build:
-    docker:
-      - image: circleci/ruby:2.6.3-stretch-node
```

Comment:
The gem supports ruby 2.6+; if we are doing a change we should add more ruby images, but not what carwow chose for internal development.